tests/acceptance/blocks/filterboxcomponent.js

```diff
@@ -0,0 +1,88 @@
+import { Selector, t } from 'testcafe';
+import FilterOptionsComponentBlock from './filteroptionscomponent';
+/**
+ * This class models user interactions with the {@link FilterBoxComponent}.
+ */
+export default class FilterBoxComponentBlock {
+  constructor (container) {
+    this._container = container || '';
+  }
+
+  /**
+   * Toggle the given option for the given filter.
+   */
+  async toggleOption (filterIndex, optionIndex) {
```

Comment:
Would it make more sense for toggle to be an option on the FilterOptions block and the FilterBox block has a method to get a FilterOptions instance by name?

Reply:
the name for a FilterOptions is `${this.name}.filter${i}` where this.name is the FilterBox name, and the container div for FilterOptions (in a FilterBox) is `.js-yext-filterbox-filter${i}`, both of which are derived from the index, not the name. I think it would make the tests a good bit more readable if we could assign names to the FilterOptions, but it seems difficult with how FilterBox currently works

Reply:
Ah nevermind I think you probably just meant by things like the title headers or other text on the screen right? That makes a lot of sense